mediathread/mixins.py

```diff
@@ -185,7 +185,8 @@ def dispatch(self, *args, **kwargs):
 class ProjectEditableMixin(object):
     def dispatch(self, *args, **kwargs):
         project = get_object_or_404(Project, pk=kwargs.get('project_id', None))
-        if not project.can_edit(self.request.course, self.request.user):
+        if (project and
```

Comment:
With a `get_object_or404()` on the previous line, the extra check here seems redundant.